index.html

```diff
@@ -250,7 +267,10 @@
         togglemathjax(latexenabled);
         e.preventDefault();
         return false;
-    }  
+    }
+    else if (e.keyCode == 79 && e.ctrlKey && e.shiftKey) {    // CTRL + SHIFT + O
+        $('#openFileInput').click();
+    } 
     else if (e.keyCode == 27)  { // ESC

```

Comment:
Same thing for this key value 27. Create a constant that has the name ESC_KEYCODE or something like that.